docs/plot.py

```diff
@@ -0,0 +1,17 @@
+import matplotlib.pyplot as plt
+from astropy.coordinates import SkyCoord
+
+from hips.draw.simple import make_sky_image
```

Comment:
Import from the top-level `hips` namespace here and on the line below.
The basic idea is to always import from the location where something appears in the API docs, i.e. where users might find it if they look at the docs.